poms/bom/pom.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.datastax.oss.quarkus</groupId>
+        <artifactId>cassandra-quarkus-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>cassandra-quarkus-bom</artifactId>
+    <packaging>pom</packaging>
+
+    <name>Cassandra Quarkus :: BOM</name>
+
+    <properties>
+        <cassandra-quarkus.version>1.0.0-SNAPSHOT</cassandra-quarkus.version>
+        <datastax-java-driver.version>4.5.0</datastax-java-driver.version>
+        <lz4.version>1.6.0</lz4.version>
+        <snappy.version>1.1.7.3</snappy.version>
+        <test-containers.version>1.12.4</test-containers.version>
+        <awaitility.version>4.0.2</awaitility.version>
+        <assertj.version>3.15.0</assertj.version>
+        <mockito.version>3.3.0</mockito.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <!-- External BOM -->
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-bom</artifactId>
+                <version>${quarkus.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+
+            <!-- Cassandra Quarkus -->
+            <dependency>
+                <groupId>com.datastax.oss.quarkus</groupId>
+                <artifactId>cassandra-quarkus-client</artifactId>
+                <version>${cassandra-quarkus.version}</version>
+            </dependency>
+
+            <!-- Other third party dependencies here -->
+            <dependency>
```

Comment:
Quarkus uses BOMs a lot, why don't we use the driver BOM as well instead of declaring the driver dependencies one by one?

Reply:
We would still need to do exclusions, so we would need to redefine them. 
I don't know if this will be beneficial, wdyt?

Reply:
I don't think the two are incompatible: the BOM only sets versions, you still need to declare (version-less) dependencies to driver artifacts in the `<dependencies>` section; these can have exclusions.

Reply:
I've changed the pom to use the `datastax-java-driver` bom. If you are definition exclusions, the version definition needs to be repeated (but this is only one time)